### DISCUSSION TITLE
SSL Certificates are not generating for my domain

### DISCUSSION THREAD

**Discussion by Avisator1** [vercel/community#7319] [Help] [answered]
### Summary

So i have a website that i just hosted with vercel and i am running into an issue, it was first like alright i thought the cert would generate, because it said that it would, but it started spinning like that for 45 minutes and then it just randomlly stopped and i never actually got it.

i don't really know how to actually come about this, but if it interests you here is my domain name:
adiavi.com

### Example

adiavi.com

### Steps to Reproduce

I don't really know how to replicate this problem.

**Comment by pawlean** [comment, marked as answer]
Hi @Avisator1! 

I checked your domain adiavi.com and it looks like it all resolved now. 

Let me know if you have any other questions!

**Comment by themonkeyzen** [comment]
Same problem here....

**Reply by pawlean** [reply]
@themonkeyzen Can you check for clues [here](https://www.sslshopper.com/ssl-checker.html)?

**Reply by themonkeyzen** [reply]
@pawlean All the checks are okay (green). I am trying to add a subdomain to a domain that I already have registered... and it doesn't work. I have tried in another project, and it doesn't work either.
Previously, this same procedure worked without any issues. Something must have changed....

**Reply by themonkeyzen** [reply]
I found the error. My domain is .top, and it seems that there is an error with Let's Encrypt and .top domains...

https://letsencrypt.status.io/
<img width="943" alt="Captura de Pantalla 2024-07-02 a las 0 49 47" src="https://github.com/vercel/community/assets/10196724/2ca86c01-f8d5-4490-a364-ff1ce86c3730">

**Reply by pawlean** [reply]
Ah ha! Thanks for keeping us up-to-date. Has it been solved now? Let us know if there's anything else we can help with.

[a comment by an automated account is not shown]
